homeassistant/components/decora_wifi/config_flow.py

```diff
@@ -0,0 +1,127 @@
+"""Config flow for myLeviton decora_wifi."""
+
+from __future__ import annotations
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.data_entry_flow import FlowResult
+
+from .common import CommFailed, DecoraWifiPlatform, LoginFailed
+from .const import CONF_TITLE, DOMAIN
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class DecoraWifiConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a Decora Wifi config flow."""
+
+    VERSION = 1
+
+    def __init__(self):
+        """Initialize the config flow class."""
+        self.data = {}
+        super().__init__()
+
+    async def async_step_import(self, data=None) -> FlowResult:
+        """Handle import from yaml."""
+        if data is not None:
+            return await self.async_step_user_validate(data)
+        return self.async_abort(reason="import_failed")
```

Comment:
```suggestion
        await self.async_step_user_validate(data)
```

data cannot be none since the only call is from `async_setup_platform` in `light.py`